src/pages/PassportRedemption/LyftPromo.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TitleRow, Title, SubTitle, Button } from './style';
+import BikeImg from './Citibike_Lyft.png';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+    isSuccess: boolean;
+}
+
+export const LyftRewardPromo = () => {
+    const { t } = useTranslation();
+    return (
+        <LyftPromptContainer height={340}>
+            <LyftIcon src={BikeImg} />
+            <TitleRow>
+                <SubTitle bold="700">
```

Comment:
could we give this a classname and bold it in the CSS?

Reply:
Done